### ISSUE TITLE
A Sticker for Activate Mozilla Campaign. 

### ISSUE THREAD

**Issue by umeshagarwal** [mozilla/OpenDesign#85]
## Goal:

A logo for round Sticker for Activate Mozilla Campaign. 
https://activate.mozilla.community
## Info:

It will be used as a swag for events happening in India.
## Style Information:

It should follow the Mozilla Style guide and should reflect (Activate Mozilla Campaign + India). Basically a localised logo for Indian audience to describe Activate Mozilla Campaign. 
https://activate.mozilla.community
## Deadline:

By 11 September as this is kind of urgent. 
## Tag:

@parag9d @brianking @elioqoshi

**Comment by parag9d** [comment]
![activate mozilla sticker-02](https://cloud.githubusercontent.com/assets/6411097/18232809/56514400-72f5-11e6-8b65-c41ee599ae18.png)

**Comment by umeshagarwal** [comment]
@parag9d  can we use the actual mozilla font ?

**Comment by parag9d** [comment]
@umeshagarwal other than that anything else?

**Comment by umeshagarwal** [comment]
For me rest looks cool! but will wait for @brianking @geroter  feedback ...

**Comment by parag9d** [comment]
![activate mozilla sticker-02](https://cloud.githubusercontent.com/assets/6411097/18232970/b85e3704-72f8-11e6-8555-d77759eb624c.png)

**Comment by parag9d** [comment]
![activate mozilla sticker-02](https://cloud.githubusercontent.com/assets/6411097/18232973/d9d28b2e-72f8-11e6-8654-8a1d4fdd49f1.png)

**Comment by parag9d** [comment]
![activate mozilla sticker-03](https://cloud.githubusercontent.com/assets/6411097/18232985/13a89a14-72f9-11e6-8ff1-0c1c47ddd1ee.png)

**Comment by umeshagarwal** [comment]
@parag9d  this looks perfect to me! I would take that last feedback from @brianking so that we can proceed ahead with sticker production. 

Thanks
Umesh Agarwal

**Comment by brianking** [comment]
I like it a lot, but I need to get feedback from a couple of people before we proceed. Will comment back here when ready. Thank you.

**Comment by umeshagarwal** [comment]
This is supposed to be used only for India. If you want we can make it the official for whole campaign.

**Comment by umeshagarwal** [comment]
@brianking as its just for India so can we move ahead as we need to get it printed soon so that we can complete our swag box.

**Comment by parag9d** [comment]
@umeshagarwal The source files of the sticker are here: https://github.com/mozilla/OpenDesign/tree/master/Activate%20Mozilla%20Sticker%20Design

**Comment by hossainalikram** [comment]
Hello @parag9d ,

The design looks awesome, but to be used as a global community resource, can you change this portion, please? This resembles to India's Flag, and it would be great, removing the flag or use something general for the global mozilla community.

![da5e0f62-72fa-11e6-92ae-cfab2a671f71](https://cloud.githubusercontent.com/assets/8605491/18612326/e3173308-7d78-11e6-9d42-2ed0a99402d7.png)

**Comment by hossainalikram** [comment]
The unassigned was not done intentionally, (Neither, I do have rights to do that in this repostitory, no idea how it happened) Sorry for that @parag9d

**Comment by parag9d** [comment]
Hi @hossainalikram This is still not finalized for global use. @brianking is looking into it. Currently, it is only aimed at the India campaign. But if in future we decide to go forward with it for the global community, we can do something like this...or maybe use flag colours of different countries for country-specific campaigns.

![activate mozilla sticker-02](https://cloud.githubusercontent.com/assets/6411097/18612352/42c37248-7d75-11e6-9bb9-77086b66d659.png)

**Comment by elioqoshi** [comment]
![activate mozilla_sticker_print-01](https://cloud.githubusercontent.com/assets/5436686/18614619/230bf6ba-7d92-11e6-8ff9-2b8343e575c3.png)

Thanks for working on this! I suggest a few changes, including:

- A solid colored sky might look better and offer more contrast
- Mozilla shouldn't be written in Fira Sans when the wordmark is needed. The classic wordmark should be used.
- The fox head should be 180' rotated, as a rocket is standing up, not down
- I muted the blue colors a bit, might be a bit friendlier to the eye

Not sure if that's on time for India now, but I'd kindly ask @umeshagarwal to refreain from such tight deadlines in the future as this is really hard to follow up with everyone having their daily jobs and stuff. Luckily @parag9d chimed in but that's not always possible. Maybe we should add a timeframe of 3 weeks in advance at least, just like a budget request at Reps.

Anyway, good work on this!

P.S: I for some reason cannot push to the repo, anyone removed my write access to it? Couldn't upload the source file.

**Comment by AlexLakatos** [comment]
As an opinionated heavy sticker user I can't help but ask:
Could we have this as a round sticker as well, pretty please?

**Comment by parag9d** [comment]
Regarding the points raised by @elioqoshi 

1. A solid colored sky might look better and offer more contrast: I personally feel that the different shades bring out more depth in the design and show different stages of the Activate Mozilla Campaign

2. Mozilla shouldn't be written in Fira Sans when the wordmark is needed. The classic wordmark should be used: Didn't understand this point.

3. The fox head should be 180' rotated, as a rocket is standing up, not down: The fox is not standing upright inside the rocket, instead, he is flying it. So, its face should point in the direction of flight of the rocket.

4. I muted the blue colors a bit, might be a bit friendlier to the eye: Kind of looks little dull now.

And @AlexLakatos regarding round stickers, we can do something similar to the t-shirt design https://github.com/mozilla/OpenDesign/issues/86

**Comment by elioqoshi** [comment]
1: I don't think it's clear that the sky has a meaning here metaphorically speaking. It is a bit heavier on the eye with many colors. I think a solid color is best (or even slight gradient would be better)

2: You didn't use the Mozilla logo I meant.

3: Ah okay, I thought it was painted on the rocket, not a window. Not sure if that gets across that it is a window? Even when it is like that, it wouldn't be upside down (there are some cartoon examples like that)

4: Should be dull. Focus should be on the rocket, not the sticker borders.

My 5 cents. Great work in any case! :)

**Comment by umeshagarwal** [comment]
Hi Guys,

We requested this design for India specific events.. But as i see that its a better idea to go for globally used design. So lets finalise it soon as we are already late.

**Comment by umeshagarwal** [comment]
I will have to go with this design for events in India.. We have a lot of production work pending.. Please continue to get a globally used design if its needed..
![da5e0f62-72fa-11e6-92ae-cfab2a671f71](https://cloud.githubusercontent.com/assets/3319062/18642775/abead5ae-7ebf-11e6-9076-13fe15ee5bcb.png)

**Comment by hossainalikram** [comment]
I am also going for production with Round sticker design ( The one which was used for T Shirt).

Thanks @parag9d for providing the assets!

**Comment by PattanAsifKhan** [comment]
![img_20170108_003227](https://cloud.githubusercontent.com/assets/22373570/21744418/4f3b0b3a-d53b-11e6-9bdb-5353b4a484fd.jpg)

**Comment by PattanAsifKhan** [comment]
![img_20170108_003211](https://cloud.githubusercontent.com/assets/22373570/21744452/0c335dd2-d53c-11e6-87b0-aaafb486ae3d.jpg)

**Comment by elioqoshi** [comment]
@PattanAsifKhan ?

**Comment by PattanAsifKhan** [comment]
Yes this is Asif 😇😇

**Comment by elioqoshi** [comment]
I mean, what was that for?

**Comment by PattanAsifKhan** [comment]
It was for Activate Mozilla Campaign. My bad,  I am not good at drawing
but,  I can explain my drawing. The drawing shows the Firefox and Human
shaking hands. That is,  The human has been influenced by the fox and the
human shakes hand with the fox confirming the fox that he is in. Deal is
done.  I hope you get it. Sorry for my bad drawing.
